src/javascripts/components/tabs.js

```diff
@@ -82,7 +92,7 @@
       $('.app-c-link--close').click(GOVUK.tabs.clickCloseContainer)
 
       // Show open containers
-      $('.app-c-tabs__heading a[open]').click()
+      $('.js-open').click()
```

Comment:
Is this scoped to the current instance of the component? Is this class too generic?

Reply:
nope, not scoped. may be too generic indeed. I'll need to have a chat about it.

Reply:
thanks everyone. I'll refactor the code to use `js-` prefixes as the [styleguide](https://github.com/alphagov/styleguides/blob/master/js.md#html-class-hooks) recommends and we'll review this afterwards.